### DISCUSSION TITLE
Inbound API v2024-03-20 is not WORKING, multiple issues with no answers from the development team.

### DISCUSSION THREAD

**Discussion by alexartisans** [amzn/selling-partner-api-models#3844] [General]
1. Expiration dates. Generally there are two workflows with "the box packaging information known" or not. We choose that we "don't know packaging options" as we cant always predict boxes content. So box content labels are printed on 2d labels as well as EXPIRATION DATES.  ItemInput object has "expiration" set as optional. BUT when we try to create plan with expiration items - we get this error:  "severity": "ERROR",
      "code": "FBA_INB_0180",
      "details": "There's an input error with the resource 'XXX-798227'.",
      "message": "ERROR: Expiration date required"
Can we create "dummy expiration dates" while we will have real expiration dates encoded onto 2d barcodes ??? Is there a call planned where we can update "expiration dates" ? Can we get an access to it ? If we cant do that - that's another "double process" that will increase cost of processing for us.
2. This was reported multiple times with 0 updates. ItemInput should have a property like the v0 version InboundShipmentPlanRequestItem.PrepDetailsList so we can pass the propType property.
BTW Item (not ItemInput) has property prepInstructions.
    "severity": "ERROR",
      "code": "FBA_INB_0182",
      "details": "There's an input error with the resource 'XXX-610280'.",
      "message": "ERROR: Prep classification for this SKU was missing. Choose the prep category that applies to this SKU and apply any required prep and labeling to each sellable unit."

There are other issues that are being reported with 0 feedback by developers team.

**Comment by johnkw** [comment]
FYI, the `FBA_INB_0182` issue is a duplicate of the earlier bug #556.
